sources/Interop/Windows/um/schannel/SCHANNEL_CERT_HASH_STORE.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/schannel.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct SCHANNEL_CERT_HASH_STORE
+    {
+        [NativeTypeName("DWORD")]
+        public uint dwLength;
+
+        [NativeTypeName("DWORD")]
+        public uint dwFlags;
+
+        [NativeTypeName("HCRYPTPROV")]
+        public nuint hProv;
```

Comment:
Just noting for myself, `HCRYPTPROV` is a typedef to `ULONG_PTR`, it isn't defined like a normal Win32 handle.